index.html

```diff
@@ -283,9 +285,21 @@ <h4>Replit</h4>
                 
                 <div class="fineprint">
                     <small class="made-by">&copy; 2023 | Maria Scholz</small>
-                    <small class="version">v1.01</small>
+                    <small class="version">v1.02</small>

```

Comment:
Figured I'd introduce you to [Semantic Versioning](https://semver.org/). You don't have to use it, but it's pretty standard in the industry, and it's worth knowing about.

Basically it's 3 numbers: MAJOR.MINOR.PATCH (i.e. 1.0.0 or 12.34.123)

When you make a change, you'll increment the numbers as follows:

- When you make bug fixes (or in practice very small functionality changes) that don't change really introduce big functionality changes to your code: Increment the PATCH number and leave the other numbers the same.
- When you make functionality changes to your application that aren't crazy big changes or shifts (and specifically changes that are all "backwards compatible" and doesn't require any consumer applications that use you to update themselves), Update the MINOR number, and reset the PATCH number to 0. 
- When you release a large major update, or anytime you release non-backwards compatible changes that would require dependent applications to update their code to maintain functionality, you increment the MAJOR number and then reset the others to 0. 

Usually you start with 0.0.1 and work your way up to v1.0.0 which is your MVP that you "officially launch with", although a lot of time people "launch" before then to solicit extra feedback from "beta" users. 

Given a version number , increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes